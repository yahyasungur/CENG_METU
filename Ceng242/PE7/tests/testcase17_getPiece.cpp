#include "submain.h"

int main() {

	Puzzle* puzzle = new Puzzle(4, 2, 2);
	
	try {
		int location[2] = {3, 3};
		const Piece& piece = puzzle->getPiece(location);
	}
	catch(exception& e) {
		cout << e.what() << endl;
	}

	delete puzzle;

	return 0;
}