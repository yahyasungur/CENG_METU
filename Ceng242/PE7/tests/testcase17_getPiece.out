The cell that you request is empty!
